packages/cli/src/oclif/commands/pull.js

```diff
@@ -0,0 +1,55 @@
+const ZapierBaseCommand = require('../ZapierBaseCommand');
+const { downloadSourceZip } = require('../../utils/api');
+const {
+  ensureDir,
+  copyDir,
+  makeTempDir,
+  removeDir,
+  validateFileExists,
+} = require('../../utils/files');
+const AdmZip = require('adm-zip');
+const colors = require('colors/safe');
+const constants = require('../../constants');
+const { listFiles } = require('../../utils/build');
+const { deleteUnignoredFiles } = require('../../utils/pull');
+const { buildFlags } = require('../buildFlags');
+
+class PullCommand extends ZapierBaseCommand {
+  async perform() {
+    if (
+      !(await this.confirm(
+        colors.yellow(
+          'This will overwrite your local integration files with the latest version. Continue?'
```

Comment:
This command is destructive. I'd expect it to ask for confirmation on which file will be overwritten or deleted before proceeding.

For example, if you run `zapier init` twice with the same directory, you will get this:

![](https://cdn.zappy.app/63439d2450487b29bbfaa08d0fb2cd9a.png)

It's done using [yeoman-generator](https://github.com/zapier/zapier-platform/blob/efe3dad7806b57e3a732a595ee744f2340f4e890/packages/cli/src/generators/index.js#L5). The library provides an ["in-memory" file system](https://yeoman.io/authoring/file-system#:~:text=An%20%E2%80%9Cin%20memory%E2%80%9D%20file%20system), which has helped us do the "ask for confirmation" thing. Can we leverage it here too?